tests/integration/test_integration.py

```diff
@@ -472,79 +471,89 @@ def test_variant_manager(app: FastAPI):
         },
     )
     variant_id = res.json()
-    assert res.status_code == 500
+    assert res.status_code == 200
 
     res = client.get(
         f"/v1/studies/{base_study_id}/variants",
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    # assert len(res.json()) == 1
-    assert res.status_code == 500
+    assert len(res.json()) == 1
+    assert res.status_code == 200
 
     res = client.post(
         f"/v1/studies/{variant_id}/commands",
-        json=[{"action": "CREATE_AREA", "args": {"name": "testZone"}}],
+        json=[
+            {
+                "action": "create_area",
+                "args": {"area_name": "testZone", "metadata": {}},
+            }
+        ],
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    assert res.status_code == 500
+    assert res.status_code == 200
 
     res = client.post(
         f"/v1/studies/{variant_id}/commands",
-        json=[{"action": "CREATE_AREA", "args": {"name": "testZone2"}}],
+        json=[
+            {
+                "action": "create_area",
+                "args": {"area_name": "testZone2", "metadata": {}},
+            }
+        ],
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    assert res.status_code == 500
+    assert res.status_code == 200
 
     res = client.get(
         f"/v1/studies/{variant_id}/commands",
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    # assert len(res.json()) == 2
-    assert res.status_code == 500
+    assert len(res.json()) == 2
+    assert res.status_code == 200
 
-    # command_id = res.json()[1]["id"]
-    command_id = "someid"
+    command_id = res.json()[1]["id"]
+    # command_id = "someid"
     res = client.put(
         f"/v1/studies/{variant_id}/commands/{command_id}?index=0",
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    assert res.status_code == 500
+    assert res.status_code == 200
 
     res = client.get(
         f"/v1/studies/{variant_id}/commands",
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    # assert res.json()[0]["id"] == command_id
-    assert res.status_code == 500
+    assert res.json()[0]["id"] == command_id
+    assert res.status_code == 200
 
     res = client.delete(
         f"/v1/studies/{variant_id}/commands/{command_id}",
         headers={
             "Authorization": f'Bearer {admin_credentials["access_token"]}'
         },
     )
-    # assert len(res.json()) == 1
-    assert res.status_code == 500
+    # assert len(res.json()) == 1 TODO: remove this line ? delete returns nothing
```

Comment:
yes delete this line, but you could make another call to list commands or get command to assert that the command is really gone